build/translations.js

```diff
@@ -0,0 +1,27 @@
+const { Document } = require("../content");
+const { VALID_LOCALES } = require("../libs/constants");
+
+function findDocumentTranslations(document) {
+  const translations = [];
+
+  for (const locale of VALID_LOCALES.values()) {
+    if (document.metadata.locale === locale) {
+      continue;
+    }
+    const translatedDocumentURL = document.url.replace(
+      `/${document.metadata.locale}/`,
+      `/${locale}/`
+    );
+    const translatedDocument = Document.findByURL(translatedDocumentURL);
+    if (translatedDocument) {
+      translations.push({
+        locale,
+        title: translatedDocument.metadata.title,
+        url: translatedDocument.url,
+      });
+    }
+  }
+  return translations;
+}
+
+module.exports = { findDocumentTranslations };
```

Comment:
This file is only ever used from the `server/index.js`. Should it live there? 
Or, we can take this function and extend your existing `./content/translations.js` file. 
And if we do that, we can rename it to something more explicit, like `findDocumentTranslationsByURL`